airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/users.json

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
Missing fields such as invitationDisabled see https://docs.opsgenie.com/docs/user-api

Reply:
I've reviewed the schema vs the "Create User" API, and can't see anything missing?

Reply:
@adamschmidt ignore :)

Reply:
Please mark as resolved